gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-6.1.1-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-6.5-all.zip
```

Comment:
do we really want to update this in the feature branch? i would prefer to change this in its own branch, to (if something is not working anymore) we can decide to not yet update.

Reply:
no idea where this came from, removed it